### ISSUE TITLE
Running MyEtherWallet Locally

### ISSUE THREAD

**Issue by MysticRyuujin** [MyEtherWallet/MyEtherWallet#2541]
- **I'm submitting a ...**

  - [x] Feature request
  - [x] Bug report

- **Feature Request** & **Bug Report**

I'm one of those weirdos who likes to run my systems locally when I can, you know, decentralization and all that 😄 and I've noticed recently that MyEtherWallet has become not so friendly to doing so.

1. There is the whole twitter situation, the popup that comes with the new version doesn't make sense to exist in the download release.

2. When you launch MyEtherWallet locally, even over HTTPS it tries to reach back to some API endpoints that are CORS enforced. Which leads to all kinds of "Network Error" popups.

![image](https://user-images.githubusercontent.com/8877131/89841935-d87e5c00-db39-11ea-9100-77c2d2034536.png)

![image](https://user-images.githubusercontent.com/8877131/89842046-21361500-db3a-11ea-9e4c-1bb43bd0cc2c.png)

3. It no longer appears to be possible to use my own RPC endpoints.

So basically, MEW used to be pretty good about being able to be ran locally, or hosted on my own web server...now...not so much.

Is this a general change in direction for MEW or are these just bugs that have not been reported? Should I give up on the idea of running MEW locally? I have no idea where the MEW team stands on this issue so I thought I'd open an issue for discussion.

**Comment by MysticRyuujin** [comment]
I did notice that when exporting settings there is a field for `"customNetworks": []` and maybe I just need to do some extra work around the import/export settings of MEW that I didn't have to do before...if that's the case, that's cool too.
